host/xtest/regression_1000.c

```diff
@@ -30,6 +30,7 @@
 #include <ta_sims_keepalive_test.h>
 #include <ta_concurrent.h>
 #include <ta_tpm_log_test.h>
+#include <ta_bti.h>
```

Comment:
Minor: `bti` out of context sounds mysterious. `<ta_arm_bti.h>` perhaps? Other occurrences in the test files are probably OK though.